DQM/SiPixelMonitorClient/src/SiPixelEDAClient.cc

```diff
@@ -96,6 +96,7 @@ SiPixelEDAClient::SiPixelEDAClient(const edm::ParameterSet &ps) {
   sipixelDataQuality_ = new SiPixelDataQuality(offlineXMLfile_);
 
   inputSourceToken_ = consumes<FEDRawDataCollection>(ps.getUntrackedParameter<string>("inputSource", "source"));
+  cablingMapToken_ = esConsumes<SiPixelFedCablingMap, SiPixelFedCablingMapRcd, edm::Transition::EndLuminosityBlock>();
```

Comment:
@makortel, here the token is defined with the EndLuminosityBlock transition because the event setup is accessed in `SiPixelEDAClient::dqmEndLuminosityBlock(...)`. However, the `edm::ESHandle` assigned in the `SiPixelEDAClient::dqmEndLuminosityBlock(...)` is later also used in `SiPixelEDAClient::dqmEndJob(...)`. I am not sure if this could have any unwanted implications.

Reply:
Unfortunately use of EventSetup products at `endJob()` (or at `endProcessBlockProduce()` to which `dqmEndJob()` translates now) is not allowed (framework may clean up the  object before reaching that point).

Best option would be to move all computations using `SiPixelFedCablingMap` to `dqmEndLuminosityBlock()`. If that is not possible (or feasible), I believe the best workaround would be to copy the `SiPixelFedCablingMap` (or necessary parts of it if the full object is no not needed) from `dqmEndLuminosityBlock()` to `dqmEndJob()`.

Reply:
Here is my attempt at implementing a workaround https://github.com/cms-sw/cmssw/pull/32093/commits/6c41889e53bea566e3f7adb1f88c7f712b7527a7